charts/budibase/templates/proxy-service-deployment.yaml

```diff
@@ -40,6 +40,24 @@ spec:
       - image: budibase/proxy:{{ .Values.globals.appVersion | default .Chart.AppVersion }}
         imagePullPolicy: Always
         name: proxy-service
+        livenessProbe:
```

Comment:
I think we shouldn't be experimenting with this in production - would be better to test in QA first then cherry-pick the changes over.

Reply:
Was planning on trying this in preprod just - but it makes sense if there's a hotfix or something